debian/control

```diff
@@ -23,6 +23,7 @@ Build-Depends:
  libsoup2.4-dev,
  libsystemd-dev,
  ostree,
+ ostree-tests (>= 2017.6)
```

Comment:
Nitpick: Can you leave the trailing comma on the list so we don’t get git diff noise when another dependency is added afterwards in future?